### PR TITLE
SIMSBIOHUB-388, 481: Add loading buttons to edit/create survey + project forms; Access request bug

### PR DESCRIPTION
**Pull request by curtisupshall**
## Links to Jira Tickets

- https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-388
- https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-481

## Description of Changes
- **SIMSBIOHUB-481**: Fixes an error handling bug when creating a SIMS Access Request.
- **SIMSBIOHUB-388**: Uses the `LoadingButton` to show loading state when creating/editing a survey or project.
  - Consolidates the edit and create forms for projects and surveys into a single component (`EditProjectForm` and `EditSurveyForm`, respectively)

## Testing Notes

- You should be able to:
  - [ ] Create a project
  - [ ] Edit a project
  - [ ] Create a survey
  - [ ] Edit a survey 
  - [ ] Send an access request successfully, even if GC Notify environment variables are not configured.

### CODE REVIEW

File: api/src/paths/administrative-activity.ts [hunk 1]
```diff
@@ -143,7 +143,12 @@ export function createAdministrativeActivity(): RequestHandler {
 
       await connection.commit();
 
-      await administrativeActivityService.sendAccessRequestNotificationEmailToAdmin();
+      try {
+        await administrativeActivityService.sendAccessRequestNotificationEmailToAdmin();
+      } catch (error) {
+        // If email notifiation fails to send, simply log the error and continue.
+        defaultLog.error({ label: 'administrativeActivity', error });
```
Comment (NickPhura) [inline api/src/paths/administrative-activity.ts]: nit: can you set `message: 'createAdministrativeActivity'` in the object.

**Review by al-rosenthal — COMMENTED** [review]
I tested this locally and works. Looks like it needs lint fix and some tests are still broken

File: api/src/paths/administrative-activity.ts (hunk 1 above)
Comment (curtisupshall) [reply api/src/paths/administrative-activity.ts]: Done.

**Review by al-rosenthal — DISMISSED** [review]
☕

**Review by NickPhura — DISMISSED** [review]
⛽

**Review by NickPhura — APPROVED** [review]

**Review by al-rosenthal — APPROVED** [review]
🎟️

**Pull request merged**